### DISCUSSION TITLE
CentOS errors on Terraform

### DISCUSSION THREAD

**Discussion by belloblj** [learndevopsonline/learndevopsonline#1554] [Q&A]
I have been getting this error when I enter terraform plan command.

Please advise on a possible solution.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/85739456/b6488f2d-072e-4738-94ae-6e429f78896b)

**Comment by belloblj** [comment]
This is my code:
![image](https://github.com/learndevopsonline/learndevopsonline/assets/85739456/b1581834-492a-4938-aa03-007ff1fb600c)

**Comment by belloblj** [comment]
Finally got this resolved.

Make sure you update the AMI owner ID always as it is not static, it changes.
you can check the launch instance page.
filter for devops-practice
select the community AMI
check for the AMI you have chosen (in my case, it was CentOS8)
input the right owner ID.

**Reply by sreedharm07** [reply]
ok
